<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addConToSci</class>
 <widget class="QDialog" name="addConToSci">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>737</width>
    <height>277</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Connection</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLineEdit" name="input_consci_search">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid #333;
padding: 5px;</string>
     </property>
     <property name="placeholderText">
      <string>Search for avalible computers...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="table_consci_connections">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>140</number>
     </attribute>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Type</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>If Made</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Year Made</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string/>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="button_consci_connect">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 0px solid #333;
padding: 7px 0 5px 0;
background-color: #5EC748;
color: #fff;
font: 13pt &quot;Helvetica&quot;;</string>
     </property>
     <property name="text">
      <string>Connect selected computer</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="button_consci_cancel">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 0px solid #333;
padding: 7px 0 4px 0;
background-color: #E94949;
color: #fff;
font-weight: bold;
font: 25 13pt &quot;Helvetica&quot;;</string>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
